sdk/src/testIntegration/java/com/hedera/hashgraph/sdk/test/integration/TokenRejectIntegrationTest.java

```diff
@@ -632,9 +632,24 @@ void canRemoveAllowanceWhenExecutingTokenRejectForFtAndNft() throws Exception {
                 .sign(spenderAccountKey)
                 .execute(testEnv.client)
                 .getReceipt(testEnv.client);
-        }).withMessageContaining("SPENDER_DOES_NOT_HAVE_ALLOWANCE");
+        }).withMessageContaining("INSUFFICIENT_TOKEN_BALANCE");
 
-        // same test for nft
+        new TokenDeleteTransaction()
+            .setTokenId(ftTokenId)
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client);
+
+        testEnv.close(receiverAccountId, receiverAccountKey);
+    }
+
+    @Test
+    @DisplayName("Can remove allowance when executing TokenReject transaction for NFT")
```

Comment:
```suggestion
    @DisplayName("Cannot remove allowance when executing TokenReject transaction for NFT")
```

Reply:
@agadzhalov its because error I received in this test is `SPENDER_DOES_NOT_HAVE_ALLOWANCE`, so I left `can` on purpose.